src/zulip.rs

```diff
@@ -347,22 +347,49 @@ impl Recipient<'_> {
                 let mut encoded_topic = Vec::new();
```

Comment:
```suggestion
                let mut encoded_topic = String::new();
```

And `use std::fmt::Write as _;` (to make `write!` work) and you can avoid the `String::from_utf8` below.

Reply:
Thanks